app/clients/sms/aws_pinpoint.py

```diff
@@ -33,6 +34,8 @@
 from app.exceptions import InvalidProviderException
 from app.feature_flags import FeatureFlag, is_feature_enabled
 
+PINPOINT_SMS_VOICE_V2_CONFIGURATION_SET_NAME = os.getenv('PINPOINT_SMS_VOICE_V2_CONFIGURATION_SET_NAME')
```

Comment:
This is really something that should be set in `app/config.py` and then passed using app.config into AwsPinpointClient when init_app is called. We can merge this as-is to get things moving, but we should fix it in the next ticket.